src/execute_lua_script.cc

```diff
@@ -32,34 +32,49 @@
 
 namespace task {
 
-std::variant<sol::object, std::string>
-execute_lua_script(sol::state& lua, sol::string_view script)
+namespace {
+
+const std::string anchor = u8"\u2693";
+constexpr gul14::string_view chunk_prefix{ u8"[string \"\u2693\"]:" };
+
+std::string process_msg(gul14::string_view msg)
 {
-    const std::string anchor = u8"\u2693";
-    constexpr gul14::string_view chunk_prefix{ u8"[string \"\u2693\"]:" };
+    // If C++ code is called by Lua and throws an exception that is not derived
```

Comment:
Hmm, the following changes I don't understand. Maybe some further comment on this?

Reply:
`anchor`, `chunk_prefix`, and `process_msg()` used to be local variables in `execute_lua_script()`. Now we also need them in `check_lua_script()`, so I extracted them into an anonymous namespace.

Reply:
Got it.

Reply:
Wait a minute - but why did you remove the lambda, i.e. `auto process_msg = [chunk_prefix)(gul14::string_view msg) -> std::string {...`?

Reply:
I just replaced the lambda by a "normal" function.